src/packagedcode/cargo.py

```diff
@@ -107,18 +108,90 @@ def build_package(package_data):
 
     name = package_data.get('package').get('name')
     version = package_data.get('package').get('version')
+    description = package_data.get('package').get('description')
```

Comment:
IMHO assign `package_data.get('package')` to a variable rather than calling it over and over